services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2437,9 +2463,9 @@ private boolean skipMigratingVersion(String clusterName, String storeName, int v
     StoreConfig storeConfig = storeConfigRepo.getStoreConfigOrThrow(storeName);
     String destCluster = storeConfig.getMigrationDestCluster();
     if (clusterName.equals(destCluster)) {
-      String versionTopic = Version.composeKafkaTopic(storeName, versionNumber);
+      PubSubTopic versionTopic = pubSubTopicRepository.getTopic(Version.composeKafkaTopic(storeName, versionNumber));
```

Comment:
We should also consider adding `PubSubTopicRepository::getVtTopic(String storeName, int version)`